plugins/nextjs/src/pageGenerator/index.ts

```diff
@@ -30,6 +30,11 @@ async function generatePage(inputOptions: Options) {
       name: 'itemName',
       message: 'Page name? (e.g. "SettingsPage")',
     },
+    {
+      type: 'input',
+      name: 'parentDir',
```

Comment:
I think rootDir or cwdDir might be better names for this as we already have dirInWorkspace that talks about a parent directory